Documentation/guides/messages/javac0000.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Xamarin.Android error/warning JAVAC0000
```

Comment:
This could optionally say "error" instead of "error/warning" since at the moment JAVAC0000 is only associated with errors. But it's fine to leave it as "error/warning" too.